impl.go

```diff
@@ -40,8 +40,26 @@ func findInterface(iface string) (path string, id string, err error) {
 		return "", "", fmt.Errorf("couldn't parse interface: %s", iface)
 	}
 
+	var importPath string
+	p := strings.Split(iface, "/")
+	if n := len(p); n > 0 {
+		lp := strings.Split(p[n-1], ".")
+		importPath = strings.Join(p[0:n-1], "/") + "/" + lp[0]
+		iface = p[n-1]
```

Comment:
At this point, given that the user has explicitly signaled their intent, I think we should go ahead and return a path and id, and not bother invoking goimports. If the user has a typo, better to fail than to have goimports (surprisingly) find a totally different matching package. Then the original goimports src setup below can remain as it was.